contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -2707,6 +2708,17 @@ TdsSendPlpDataHelper(char *data, int len)
 	return rc;
 }
 
+static int
+TdsSendVarlenDataHelper(char *data, int len) {
+	int			rc;
+	if ((rc = TdsPutInt16LE(len)) == 0)
+	{
+		TdsPutbytes(data, len);
+		TdsSendDone(TDS_TOKEN_DONE, TDS_DONE_COUNT, TDS_CMD_SELECT, 1);
+	}
+	return rc;
+}
```

Comment:
This intends to implement the following from [2.2.5.2.3 Data Type Dependent Data Streams](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/3f983fde-0509-485a-8c40-a9fa6679a828) without using PLP (that is not supported in JTDS): 

> For variable-length types, with the exception of PLP (see Partially Length-prefixed Bytes below), the TYPE_VARLEN value defines the length of the data to follow.
> [...]
> The data following this can be a stream of bytes or a NULL value.